README.md

```diff
@@ -114,6 +114,14 @@ This produces a `bin` folder containing the `flow` binary.
 
 *Note: at this time, the OCaml dependency prevents us from adding Flow to [npm](http://npmjs.org). Try [flow-bin](https://www.npmjs.org/package/flow-bin) if you need a npm binary wrapper.*
 
+Flow can also compile its parser to JavaScript. [Read how here](src/parser/README.md).
+
+## Using Flow's parser from JavaScript
+
+While Flow is written in OCaml, its parser is available as a compiled-to-JavaScript module published to npm, named [flow-parser](https://www.npmjs.com/package/flow-parser). **Most end users of Flow
+will not need to use this parser directly** (and should install [flow-bin](https://www.npmjs.org/package/flow-bin) from npm above), but JavaScript packages which make use of parsing
+Flow-typed JavaScript can use this to generate Flow's syntax tree with inferred types attached.
```

Comment:
I'm afraid this isn't quite true. The parser will output an AST with Flow type annotations included, but inferred types are not represented in the AST. The `flow-parser` npm package contains only the Flow parser.